spec/features/admin/budget_investments_spec.rb

```diff
@@ -532,14 +543,14 @@
       visit admin_budget_budget_investments_path(@budget)
       within('#filter-subnav') { click_link 'Selected' }
 
-      expect(page).to have_content('There is 1 investment')
+      expect(page).to have_content('There are 2 investments')
```

Comment:
This change and the next one its because the whole context uses a @budget reference :(. Some day I'll go down the specs folder and make each scenario as self-sufficient and isolated as possible.